README.md

```diff
@@ -135,7 +137,136 @@ each element up to 255 chars.  For example:
 
 ```go
 myAAD := []string{"First aad string", "second aad string", "third aad string"}
-myDEK := []byte{"thisisadataencryptionkey"}
+myDEK := []byte{"NWvfrThUqP9aFmTWFgB86qztK2BuN0qIGg7K7kcCCRs="}
+// Do some encryption with myDEK
+// Wrap the DEK so we can safely store it
+wrappedDEK, err := client.Wrap(ctx, crkID, myDEK, &myAAD)
+
+
+// Unwrap the DEK
+dek, err := client.Unwrap(ctx, crkID, wrappedDEK, &myAAD)
+// Do some encryption/decryption using the DEK
+// Discard the DEK
+dek = nil
+```
+
+Have key Protect create a DEK for you:
+
+```go
+dek, wrappedDek, err := client.WrapCreateDEK(ctx, crkID, nil)
+// Do some encrypt/decrypt with the dek
+// Discard the DEK
+dek = nil
+
+// Save the wrapped DEK for later.  Use Unwrap to use it.
+```
+
+Can also specify AAD:
+
+```go
+myAAD := []string{"First aad string", "second aad string", "third aad string"}
+dek, wrappedDek, err := client.WrapCreateDEK(ctx, crkID, &myAAD)
+// Do some encrypt/decrypt with the dek
+// Discard the DEK
+dek = nil
+
+// Save the wrapped DEK for later.  Call Unwrap to use it, make
+// sure to specify the same AAD.
+```
+
+
+# Usage With Quantum Safe Crypto
+
+IBM Cloud Key Protect Service supports quantum safe crypto (QSC) configuration as well. Visit [link to the KP Docs here] for QSC supported endpoints, algorithms and other details.
+
+**Note**: Pre-compiled package is supported on Ubuntu at this time. Other platform support will be added in future. Meanwhile to compile packages on other platforms visit documentaion [link here]
+
+Follow below steps or copy below commands to compile client with quantum config:
+
+1. Download debian package compiled with quantum configuration. For more information on the package visit [IBM oqssa repo](https://github.com/IBM/oqssa)
+  
+    curl https://api.github.com/repos/IBM/oqssa/releases/latest | grep "browser_download_url" | cut -d '"' -f 4 | wget -i -
+1. Install package. Package will downalod to /opt/oqssa directory:
+  
+    dpkg -i oqssa-0.1.0_amd64.deb
+1. Export path:
+  
+    export PATH=/opt/oqssa/:$PATH
+1. Compile client with --tags=quantum to use quantum configuration:
+
+    CPATH=/opt/oqssa/include/ PKG_CONFIG_PATH=/opt/oqssa/lib/pkgconfig LD_LIBRARY_PATH=/opt/oqssa/lib go test --tags quantum
+
+```
+curl https://api.github.com/repos/IBM/oqssa/releases/latest | grep "browser_download_url" | cut -d '"' -f 4 | wget -i -
+dpkg -i oqssa-0.1.0_amd64.deb
+export PATH=/opt/oqssa/:$PATH
+CPATH=/opt/oqssa/include/ PKG_CONFIG_PATH=/opt/oqssa/lib/pkgconfig LD_LIBRARY_PATH=/opt/oqssa/lib go test --tags quantum  
```

Comment:
Had discussion with Cameron - we are going to leave this as test since there is nothing to build in the SDK. But user can run unit test directly to see something working.